mne/io/brainvision/brainvision.py

```diff
@@ -64,12 +69,14 @@ class RawBrainVision(_BaseRaw):
     @verbose
     def __init__(self, vhdr_fname, montage=None,
                  eog=('HEOGL', 'HEOGR', 'VEOGb'), misc=(),
-                 scale=1., preload=False, response_trig_shift=0, verbose=None):
+                 scale=1., preload=False, response_trig_shift=0,
+                 event_id={}, verbose=None):
```

Comment:
I was going back and forth between the two. I just have it look for a key in the dictionary which made it easier for a `dict()` default. I could just have `if event_id is None: event_id = dict() ` within the code